src/pawelak-arek/Fsd.Pawelak.Cs/Fsd.Arek.Cs2/Domain/ConsoleMenu.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Fsd.Arek.Cs2.Domain.Enum;
+using System;
+using System.Collections.Generic;
+
+namespace Fsd.Arek.Cs2.Domain
+{
+    public class ConsoleMenu
+    {
+        const string line = "-----------------------------------";
+
+        public void Run()
+        {
+            AnimalsFactory AnimalsFactory = new AnimalsFactory();
+            SetHeader(2, line);
+
+            GetUserInformation(AnimalsFactory, AnimalsType.cat);
```

Comment:
jak będzie 30 różnych zwierzaków to napiszesz 30x GetUserInformation(...) może pętla?